docs/timeseries/index.rst

```diff
@@ -0,0 +1,333 @@
+    2009-05-02T00:43:38.045  1.0270762e+06
+    2009-05-02T00:44:36.894  1.0271414e+06
+    2009-05-02T00:45:35.752  1.0271569e+06
+    2009-05-02T00:46:34.601  1.0272296e+06
+    2009-05-02T00:47:33.451  1.0273199e+06
+    2009-05-02T00:48:32.291  1.0271497e+06
+    2009-05-02T00:49:31.149  1.0271755e+06
+                        ...            ...
+    2009-05-11T17:58:22.526  1.0234769e+06
+    2009-05-11T17:59:21.376  1.0234574e+06
+    2009-05-11T18:00:20.225  1.0238128e+06
+    2009-05-11T18:01:19.065  1.0243234e+06
+    2009-05-11T18:02:17.923  1.0244257e+06
+    2009-05-11T18:03:16.772  1.0248654e+06
+    2009-05-11T18:04:15.630  1.0250156e+06
+    2009-05-11T18:05:14.479  1.0254516e+06
+    2009-05-11T18:06:13.328  1.0254685e+06
+    2009-05-11T18:07:12.186  1.0259309e+06
+
+    >>> ts[0:4]  # doctest: +REMOTE_DATA
+    <TimeSeries length=4>
+              time             timecorr   ...   pos_corr1      pos_corr2
+                                  d       ...      pix            pix
+             object            float32    ...    float32        float32
+    ----------------------- ------------- ... -------------- --------------
+    2009-05-02T00:41:40.338  6.630610e-04 ...  1.5822421e-03 -1.4463664e-03
+    2009-05-02T00:42:39.188  6.630857e-04 ...  1.5743829e-03 -1.4540013e-03
+    2009-05-02T00:43:38.045  6.631103e-04 ...  1.5665225e-03 -1.4616371e-03
+    2009-05-02T00:44:36.894  6.631350e-04 ...  1.5586632e-03 -1.4692718e-03
+
+
+As seen in the example above, |TimeSeries| objects have a ``time``
+column, which is always the first column. This column can also be accessed using
+the ``.time`` attribute::
+
+    >>> ts.time  # doctest: +REMOTE_DATA
+    <Time object: scale='tdb' format='isot' value=['2009-05-02T00:41:40.338' '2009-05-02T00:42:39.188'
+      '2009-05-02T00:43:38.045' ... '2009-05-11T18:05:14.479'
+      '2009-05-11T18:06:13.328' '2009-05-11T18:07:12.186']>
+
+and is always a |Time| object (see :ref:`Times and Dates <astropy-time>`), which
+therefore supports the ability to convert to different time scales and formats::
+
+    >>> ts.time.mjd  # doctest: +REMOTE_DATA
+    array([54953.0289391 , 54953.02962023, 54953.03030145, ...,
+           54962.7536398 , 54962.75432093, 54962.75500215])
+
+    >>> ts.time.unix  # doctest: +REMOTE_DATA
+    array([1.24122483e+09, 1.24122489e+09, 1.24122495e+09, ...,
+           1.24206505e+09, 1.24206511e+09, 1.24206517e+09])
+
+Let's use what we've seen so far to make a plot
+
+.. plot::
+   :context: reset
+   :nofigs:
+
+    from astropy.utils.data import get_pkg_data_filename
+    filename = get_pkg_data_filename('timeseries/kplr010666592-2009131110544_slc.fits')
+    from astropy.timeseries import TimeSeries
+    ts = TimeSeries.read(filename, format='kepler.fits')
+
+.. plot::
+   :include-source:
+   :context:
+
+   import matplotlib.pyplot as plt
+   plt.plot(ts.time.jd, ts['sap_flux'], 'k.', markersize=1)
+   plt.xlabel('Barycentric Julian Date')
```

Comment:
Related to @pllim's comment: it's actually not at all obvious this time series is BJD until you say this.  I'm guessing you know that because it's that's what Kepler light curves are provided in? It might be useful to explicitly point that out somewhere.

More broadly... This raises the question: should the timeseries *themselves* store this information? Or is that too context-specific?

Reply:
So actually for the purposes of the example, it's enough to say 'Julian Date' in the labels. But I've now also added information about finding out the time scale, which you can get with ``ts.time.scale`` (in the Kepler example this returns tdb = barycentric dynamical time). Since this is stored by the time object, I'd rather not also have it in the time series object itself.